docs/Installation/Docker.md

```diff
@@ -0,0 +1,41 @@
+description: Run EthSigner from Docker image
+<!--- END of page meta data -->
+
+# Running EthSigner from Docker Image
+
+A Docker image is provided to run EthSigner in a Docker container. 
+
+## Prerequisites
+
+* [Docker](https://docs.docker.com/install/)   
+
+* MacOS or Linux 
+    
+    !!! important 
+        The Docker image does not run on Windows. 
+
+## Quickstart
+
+Display the EthSigner command line help using the Docker image: 
+
+```bash tab="latest"
+docker run pegasyseng/ethsigner:latest --help
+```
+!!! note
+    `latest` runs the latest cached version. To pull the latest version, use `docker pull pegasyseng/pantheon:latest`. 
+ 
+## Expose Listening Port
+
+Exposing the listening port is required to use the  default listening port (`8545`) or the port specified using 
+[`--http-listen-port`](../Reference/EthSigner-CLI.md#http-listen-port). 
+
+To run EthSigner exposing listening port for access: 
+```bash
+docker run -p <listenPort>:8545 pegasyseng/ethsigner:latest <options>
```

Comment:
This won't run Ethsigner unfortunately and will only display the help as signer needs to be provided. So not sure how make clear this won't actually run but it's just what you need to add to the command.

Reply:
I've used <options> to indicate that options must be specified and the example below shows options.  Do you think this is enough?  Or I need to rethink?